ui/app/templates/components/dashboard/replication-card.hbs

```diff
@@ -30,7 +32,7 @@
           @clusterStates={{cluster-states @replication.dr.state}}
         />
         <Dashboard::ReplicationStateText
-          @title="Perf primary"
+          @title="Performance primary"
```

Comment:
I would need to test this, but you can have a performance secondary and show the dashboard. Do we account for a situation where an instance of vault is a performance secondary? (DR secondary is the only time we don't show the dashboard).